### PR TITLE
Add new companies

### PR DESCRIPTION
**Pull request by mfilej**
- ARA LOGOS LOGISTICS TRUST
- AXA SA
- Africa Oil Corp
- AngloGold Ashanti Ltd
- BHP GROUP LTD
- BP Plc
- BTCS Inc
- Banco Bradesco SA
- Banco Macro SA
- Banco Santander SA
- Bank of East Asia Ltd
- Bayerische Motoren Werke AG
- British American Tobacco plc
- CNOOC Ltd
- Centamin Plc
- China Gold International Resources Corp Ltd
- China Mobile Ltd
- Cia de Minas Buenaventura SA
- Danone SA
- Deutsche Pfandbriefbank AG
- ESR-Logos REIT
- Fortitude Gold Corp
- Franco-Nevada Corporation
- GSK Plc
- Gold Resource Corp
- Hang Lung Properties Ltd
- Hecla Mining Co
- Hochschild Mining Plc
- Hutchison Telecommunications HK Hld Ltd
- IGO Ltd
- ING Groep NV
- ISHARES MSCI ITALY ETF
- ISHARES MSCI SPAIN ETF
- Imperial Brands Plc
- Indofood Agri Resources Ltd
- Itau Unibanco SA
- Keppel Infrastructure Trust
- Marriott International Inc
- Mercedes-Benz Group AG
- NWS Holdings Ltd
- Newcrest Mining Ltd
- Novartis AG
- Orange
- Osisko Gold Royalties Ltd
- Pan American Silver Corp.
- Petrochina Co Ltd
- Petroleo Brasileiro SA
- Repsol SA
- SINOPEC Engineering Group Co Ltd
- SPDR S&P Oil & Gas Equipment & Services ETF
- Sanofi SA
- Shell plc
- Sibanye Stillwater Ltd
- Singapore Communications Ltd
- SmarTone Telecommunications Holdings Ltd
- Standard Chartered PLC
- Swiss Life Holding AG
- Swiss Re AG
- Swisscom AG
- Telefónica Deutschland Holding AG
- Telefonica SA
- Telefónica, S.A.
- Turkcell İletişim Hizmetleri AS
- Veolia Environnement
- Wilmar International Ltd
- Woodside Energy Group Ltd
- Yamana Gold Inc
- Zurich Insurance Group AG
- iShares MSCI Switzerland ETF
-

### CODE REVIEW

File: companies.xml [hunk 1]
```diff
@@ -1420,4 +1428,588 @@
...
+        <name>Yamana Gold Inc</name>
+        <taxNumber>125079871</taxNumber>
+        <address>40 King Street West, Suite 2100, Toronto ON M5H 3C2</address>
+        <country>CA</country>
+    </company>
+    <company>
+        <conid>10828168</conid>
+        <symbol>PBR</symbol>
+        <name>Petroleo Brasileiro SA</name>
+        <taxNumber>33000167/0245-58</taxNumber>
```
Comment (ddbk) [inline companies.xml:1955]: According to some Brazilian docs, headquarters always have the 0001-01 ending. This is what I found:
```suggestion
        <taxNumber>33.000.167/0001-01</taxNumber>
```

File: companies.xml [hunk 2]
```diff
@@ -1420,4 +1428,588 @@
...
+        <address>Av. Henrique Valadares, 28 - Centro, Rio de Janeiro - RJ, 20231-030</address>
```
Comment (ddbk) [inline companies.xml:1956]: I found the following address in the last annual report, but noticed their address changed to what you added in the last SEC report, so I'm not sure which is actually the correct one:
```suggestion
        <address>Avenida República do Chile 65, 20031-912, Rio de Janeiro, RJ</address>
```

**Comment by jamsix** [comment]
Thanks @mfilej, @ddbk! This is huuuuge.

**Comment by pronebird** [comment]
Spent whole evening looking for Osisko's ID damn!

File: companies.xml (hunk 2 above)
Comment (ddbk) [reply companies.xml]: @mfilej I took some time to look more into this and it seems they started reporting the new address (Av. Henrique Valadares) in their SEC filings on April 13, 2022. April 12 was still the one I suggested (Av. Republica do Chile). This change coincides with the date of 2022 Annual General Meeting and election of the new CEO, but I haven't found anything related to the actual change of address. So I'm not sure what to put here. 🤷‍♂️

**Comment by mfilej** [comment]
@ddbk Thanks for the corrections!

**Pull request merged**